src/pages/GettingStarted/GettingStarted.tsx

```diff
@@ -1,20 +1,50 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
+import useFetch from 'use-http';
+import { toast } from 'react-toastify';
 import { useStyles } from '@grafana/ui';
 import { PrivateLayout } from 'components/Layouts';
 import { Routes } from 'core/routes';
 import { Messages } from './GettingStarted.messages';
 import { getStyles } from './GettingStarted.styles';
 import { GettingStartedSection } from './GettingStartedSection';
+import { LIST_ORGANIZATION_URL } from './GettingStarted.constants';
 
 export const GettingStartedPage: FC = () => {
   const styles = useStyles(getStyles);
+  const [orgIds, setOrgIds] = useState([]);
+  const { error, data = {} } = useFetch(LIST_ORGANIZATION_URL, { method: 'POST' }, []);
+
+  useEffect(() => {
+    if (error) {
+      toast.error(Messages.fetchError);
+    }
+  }, [error]);
+
+  useEffect(() => {
+    const { org_ids } = data;
+
+    setOrgIds(org_ids);
```

Comment:
```suggestion
    setOrgIds(data?.org_ids);
```
No need to destructure IMO )

Reply:
Good point, done